src/components/Eligibility.tsx

```diff
@@ -14,15 +14,15 @@ export default function Eligibility({
     case 'missing':
       return (
         <p className="rounded-lg py-5 px-6 text-base mb-3 bg-red-100 text-red-700">
-          To govern you must first have received an invitation to the Econ
+          To propose you must first have received an invitation to the Econ
           Committee Charter.
         </p>
       );
     case 'available':
       return (
         <div className="rounded-lg py-5 px-6 text-base mb-3 bg-yellow-100 text-yellow-700">
-          To vote you will need to accept your invitation to the Econ Committe
-          Charter.
+          To propose you will need to accept your invitation to the Econ
```

Comment:
the `Eligibility` component name sounds like it would encompass both vote (EC) and propose (EC Charter) cases. but it's just the latter so let's also rename it. `CharterGuidance`?

Reply:
Done